iree/test/e2e/regression/BUILD

```diff
@@ -221,3 +225,31 @@ py_binary(
 ] for size in [
     "small",
 ]]
+
+[iree_generated_trace_runner_test(
+    name = "e2e_matmul_direct_f32_gpu_large_%s" % compilation_info,
+    generator = ":generate_e2e_matmul_tests",
+    generator_args = [
+        "--lhs_rhs_type=f32",
+        "--shapes=gpu_large",
+        "--compilation_info=%s" % compilation_info,
+    ],
+    tags = [
+        # CUDA cuInit fails with sanitizer on.
+        "noasan",
+        "nomsan",
+        "notsan",
+        "noubsan",
+        "requires-gpu-nvidia",
+        "driver=cuda",
+        "nokokoro",
+        "manual",
+    ],
```

Comment:
These are only added to the test, not the test deps.

https://github.com/google/iree/blob/e10830fb3a7dba1d77fe517b9620a0a1a797f7e3/build_tools/bazel/iree_trace_runner_test.bzl#L39-L87

Reply:
Here, you want to tag the `iree_bytecode_module` as `manual` + `nokokoro` (or the cuda tags), as that requires the compiler.

Here is how the `iree_check_test` function skips cuda: https://github.com/google/iree/blob/e10830fb3a7dba1d77fe517b9620a0a1a797f7e3/build_tools/bazel/iree_check_test.bzl#L128-L132

Reply:
Another option is to just omit this particular `iree_generated_trace_runner_test` from the BUILD file and put it in the `BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE` section of the CMake file. (That will be cleaner once the `iree_generated_trace_runner_test` test are in their own dir, instead of next to the other e2e/regression/ tests)

Reply:
I've updated the `iree_trace_runner_test.bzl` to skip tests the use the cuda backend, since bazel doesn't support cuda.